app/views/people/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> People </h1>
+
```

Comment:
I might look into using a table to structure this data to be more visually readable